theme/page.template

```diff
@@ -15,7 +15,7 @@
 <![endif]-->
 <link rel="shortcut icon" href="">
 </head>
-<body>
+<body dir="<%= direction %>">
```

Comment:
Did you test it?

After adding `dir="RTL"` into the page, I got the following looking. Is it right?

![redux documents](https://cloud.githubusercontent.com/assets/905434/24803554/2f7305fc-1bde-11e7-868e-591c53ebfe6a.png)